--- origsrc/gobject-introspection-1.36.0/tests/scanner/GetType-1.0-expected.gir	2013-02-26 15:25:11.000000000 -0600
+++ src/gobject-introspection-1.36.0/tests/scanner/GetType-1.0-expected.gir	2013-08-04 21:54:43.219162200 -0500
@@ -11,7 +11,7 @@ and/or use gtk-doc annotations.  -->
   <c:include name="gettype.h"/>
   <namespace name="GetType"
              version="1.0"
-             shared-library="libgettype.so"
+             shared-library="cyggettype.dll"
              c:identifier-prefixes="GetType"
              c:symbol-prefixes="gettype">
     <class name="Object"
--- origsrc/gobject-introspection-1.36.0/tests/scanner/GtkFrob-1.0-expected.gir	2013-08-04 21:54:10.037264300 -0500
+++ src/gobject-introspection-1.36.0/tests/scanner/GtkFrob-1.0-expected.gir	2013-08-04 21:52:26.000000000 -0500
@@ -10,7 +10,7 @@ and/or use gtk-doc annotations.  -->
   <package name="gobject-2.0"/>
   <namespace name="GtkFrob"
              version="1.0"
-             shared-library="libgtkfrob.so"
+             shared-library="cyggtkfrob.dll"
              c:identifier-prefixes="Gtk"
              c:symbol-prefixes="gtk_frob">
     <function name="language_manager_get_default"
--- origsrc/gobject-introspection-1.36.0/tests/scanner/Regress-1.0-expected.gir	2013-08-04 21:54:10.052265200 -0500
+++ src/gobject-introspection-1.36.0/tests/scanner/Regress-1.0-expected.gir	2013-08-04 21:52:26.000000000 -0500
@@ -12,7 +12,7 @@ and/or use gtk-doc annotations.  -->
   <c:include name="regress.h"/>
   <namespace name="Regress"
              version="1.0"
-             shared-library="libregress.so"
+             shared-library="cygregress.dll"
              c:identifier-prefixes="Regress"
              c:symbol-prefixes="regress">
     <alias name="AliasedTestBoxed" c:type="RegressAliasedTestBoxed">
--- origsrc/gobject-introspection-1.36.0/tests/scanner/SLetter-1.0-expected.gir	2013-02-26 15:25:11.000000000 -0600
+++ src/gobject-introspection-1.36.0/tests/scanner/SLetter-1.0-expected.gir	2013-08-04 21:54:52.968719800 -0500
@@ -10,7 +10,7 @@ and/or use gtk-doc annotations.  -->
   <c:include name="sletter.h"/>
   <namespace name="SLetter"
              version="1.0"
-             shared-library="libsletter.so"
+             shared-library="cygsletter.dll"
              c:identifier-prefixes="S"
              c:symbol-prefixes="s">
     <enumeration name="DBusError"
--- origsrc/gobject-introspection-1.46.0/tests/scanner/Typedefs-1.0-expected.gir	2014-02-08 10:22:47.000000000 -0600
+++ src/gobject-introspection-1.46.0/tests/scanner/Typedefs-1.0-expected.gir	2015-11-05 19:18:42.207981400 -0600
@@ -11,7 +11,7 @@ and/or use gtk-doc annotations.  -->
   <c:include name="typedefs.h"/>
   <namespace name="Typedefs"
              version="1.0"
-             shared-library="libtypedefs.so"
+             shared-library="cygtypedefs.dll"
              c:identifier-prefixes="Typedefs"
              c:symbol-prefixes="typedefs">
     <record name="BoxedWithAnonymousTypedef"
--- origsrc/gobject-introspection-1.36.0/tests/scanner/Utility-1.0-expected.gir	2013-08-04 21:54:10.060265600 -0500
+++ src/gobject-introspection-1.36.0/tests/scanner/Utility-1.0-expected.gir	2013-08-04 21:52:26.000000000 -0500
@@ -11,7 +11,7 @@ and/or use gtk-doc annotations.  -->
   <c:include name="utility.h"/>
   <namespace name="Utility"
              version="1.0"
-             shared-library="libutility.so"
+             shared-library="cygutility.dll"
              c:identifier-prefixes="Utility"
              c:symbol-prefixes="utility">
     <alias name="Glyph" c:type="UtilityGlyph">
